src/pages/vrf/v2/lootbox.mdx

```diff
@@ -0,0 +1,267 @@
+
+[Hardhat](https://hardhat.org/) is an Ethereum development environment that is used here to configure and deploy the lootbox contract.
+
+1. In your `.env` file, fill in these values:
+
+   | Parameter         | Description                                                 | Example                                     |
+   | ----------------- | ----------------------------------------------------------- | ------------------------------------------- |
+   | `NETWORK_RPC_URL` | The RPC URL for the network you want to deploy to.          | `https://sepolia.infura.io/v3/your-api-key` |
+   | `PRIVATE_KEY`     | The private key of the account you want to deploy from.     | `0xabc123abc123abc123abc123abc123...`       |
+   | `ETHERSCAN_API`   | The API key for Etherscan needed for contract verification. | `ABC123ABC123ABC123ABC123ABC123ABC1`        |
+
+1. Configure your deployment network in [`hardhat.config.ts`](https://github.com/smartcontractkit/quickstarts-lootbox/blob/main/hardhat.config.ts).
+
+#### Contract parameters
+
+In your `.env` file, fill in these values:
+
+| Parameter                             | Description                                                                                                               | Example      |
+| ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| `LOOTBOX_FEE_PER_OPEN`                | The fee per open in ETH                                                                                                   | `0.1`        |
+| `LOOTBOX_AMOUNT_DISTRIBUTED_PER_OPEN` | The amount of reward units distributed per open                                                                           | `1`          |
+| `LOOTBOX_OPEN_START_TIMESTAMP`        | The start timestamp in UNIX time for the public open. Leave blank to start immediately.                                   | `1630000000` |
+| `VRF_SUBSCRIPTION_ID`                 | A funded Chainlink VRF subscription ID. If you leave this blank, a new subscription will be created and funded on deploy. | `123`        |
+
+#### Reward tokens
+
+For rewards, the lootbox contracts supports any tokens from the following [token standards](https://ethereum.org/en/developers/docs/standards/tokens/):
+
+- [ERC-20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/): the fungible token standard
+- [ERC-721](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/): the NFT standard
+- [ERC-1155](https://ethereum.org/en/developers/docs/standards/tokens/erc-1155/): the multi-token standard
+
+The amounts per unit are used to calculate the lootbox supply of rewards. For example, if you want to distribute 100 tokens of a specific ERC20 token, you would set the `totalAmount` to `100000000000000000000` and the `amountPerUnit` to `1000000000000000000`. This would result in a lootbox supply of 100 units.
```

Comment:
```suggestion
The amounts per unit are used to calculate the lootbox supply of rewards. For example, if you want to distribute 100 tokens of a specific ERC-20 token, you would set the `totalAmount` to `100000000000000000000` and the `amountPerUnit` to `1000000000000000000`. This would result in a lootbox supply of 100 units.
```